pkg/build/cmd/buildbackend.go

```diff
@@ -18,6 +18,7 @@ import (
 func BuildBackend(ctx *cli.Context) error {
 	metadata, err := config.GetMetadata(filepath.Join("dist", "version.json"))
 	if err != nil {
+		fmt.Println("err")
```

Comment:
debugging?

Reply:
Yep yep, the title of the commit is [[REVERTME] Partially sync version changes](https://github.com/grafana/grafana/pull/53516/commits/57a2be290bd97d6cf0960af35b6500d89d4b458f), so I'll make sure I revert that :D